tests/unit/amo/components/TestAddon.js

```diff
@@ -298,6 +298,8 @@ describe(__filename, () => {
 
     // Since withInstallHelpers relies on this, make sure it's initialized.
     expect(root.instance().props.platformFiles).toEqual({});
+
+    expect(root.find('.Addon-icon img').prop('alt')).toEqual('');
```

Comment:
Could we set the value to `null` so that it would not render an empty `alt` attribute?

Reply:
oh I thought empty attribute was better than leaving it off but sure I can update